scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+//@import 'colors';
+
+.tech {
+  width: 200px;
+  height: 300px;
+  padding: 0px 25px;
+  margin: 20px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #BED23C;
+  border-radius: 10px;
+
+  &::after {
+    content: ' ';
+    width: 200px;
+    height: 300px;
+    border: 1px solid #BED23C;
+    border-radius: 10px;
+    position: absolute;
+    top: -10px;
+    left: -10px;
+    z-index: -1;
+  }
+  
+
+  .tech-image {
+    width: 100px;
+    margin-bottom: 15px;
+  }
+
+  .tech-title {
+    margin-bottom: 15px;
```

Comment:
You are repeating the same distance between each element in the card. Maybe you can extract this size and reuse it